docs/reference/design/actions.md

```diff
@@ -59,16 +59,15 @@ the same exact path for an action that needs to respond to all those verbs in th
 same manner. There is a subtle difference, however, and that is that using 'ANY'
 will truly accept any incoming HTTP verb string, while listing them in several routes
 will need to match the specific supported names. For example, an 'ANY' route like the
-above will be able to match incoming requests like `"LINK /"` or `"UNLINK /"` (assuming the Web
-server supports it).
+above will be able to match incoming requests like `"LINK /"` or `"UNLINK /"` (assuming the Web server supports it).
 
 Remember that Praxis prefixes all your resources' routes with a string based
 on the name of your enclosing endpoint definition class, in this case
 '/blogs' since our class is called `Blogs`. You can, however, override the prefix for a single route by prepending '//' to the path (like in the example above) if you don't want the resource-wide prefix to apply. Alternately, you can provide a special prefix of either `''` or `'//'` in the routing block to clear the prefix for any other paths given.
 
-*Note*: The above 'resetting' behavior of '//' applies *only* to any Resource-level route prefixes that may be defined. It will *not* override an API-wide `base_path` if one is defined (see [Global Api Info](global-api-information/)).
+*Note*: The above 'resetting' behavior of '//' applies *only* to any Resource-level route prefixes that may be defined. It will *not* override an API-wide `base_path` if one is defined (see [Global Api Info](api-definition#global-information)).
 
-You can inspect the complete Praxis routing table using `praxis routes` or `rake praxis:routes`:
+You can inspect the complete Praxis routing table using `bundle exec praxis routes` or `bundle exec rake praxis:routes`:
```

Comment:
I am not sure if want to include `bundle exec ` here, but it doe snot work w/o it for me

Reply:
For the rake it is good to add...for the binary it shouldn't be needed...